vms/platformvm/blocks/block.go

```diff
@@ -45,3 +46,11 @@ func initialize(blk Block) error {
 	}
 	return blk.initialize(bytes)
 }
+
+func Size(blk Block) int {
+	if blk == nil {
+		return wrappers.LongLen
+	}
+
+	return wrappers.LongLen + len(blk.Bytes())
```

Comment:
Confirming it is expected this no longer has `wrappers.IntLen`

Reply:
We no longer have the `Status` in memory - so it was removed from the byte overhead